client/my-sites/plans/jetpack-plans/product-store/hooks/use-item-lightbox.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback, useMemo, useState } from 'react';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { SelectorProduct } from '../../types';
+
+export const useItemLightbox = () => {
+	const [ currentItem, setCurrentItem ] = useState< SelectorProduct | null >( null );
+	const clickMoreHandlerFactory = useCallback(
+		( item: SelectorProduct ): VoidFunction | undefined => {
```

Comment:
This no longer returns `undefined`

```suggestion
		( item: SelectorProduct ): VoidFunction => {
```

Reply:
Yes this has been updated. 👍